olive/workflows/run/run.py

```diff
@@ -213,13 +213,42 @@ def set_olive_config_for_aml_system(olive_config: dict):
     AzureMLSystem.olive_config = olive_config
 
 
+def _setup_for_winml():
+    # pylint: disable=protected-access
+    # TODO(olivedev): Remove post ORT release >1.22.0
+    try:
+        import onnxruntime as ort
+        from packaging import version
+
+        if version.parse(ort.__version__) >= version.parse("1.22.0"):
+            ort.winml.register_execution_providers()
```

Comment:
```
    import sys
    from .winml import register_execution_providers
    this = sys.modules[__name__]
    register_execution_providers(this)
```

An error occurred during WinML setup: register_execution_providers() missing 1 required positional argument: 'ort'

Reply:
but if I add it directly, it will be `An error occurred during WinML setup: module 'olive.workflows.run.run' has no attribute 'register_execution_provider_library'`

Reply:
so I guess it only workable in win-ort?

Reply:
or at least I should pass in the module of win-ort not `this`

Reply:
just debugged, please remove this line `ort.winml.register_execution_providers()`